pycaret/tests/test_time_series.py

```diff
@@ -194,7 +194,7 @@ def test_create_model(name, fh, load_data):
     )
     model_obj = exp.create_model(name)
 
-    y_pred = model_obj.predict()
+    y_pred = exp.predict_model(model_obj)
```

Comment:
I think it is good time to pass the fixtures to a `conftest.py` file and functions to other module. 

```python
for time_series_test_utils import (
    _return_model_parameters,
    _return_model_names,
    _get_seasonal_values
)
```

and 

```python
# conftest.py
import pytest

@pytest.fixture(scope="session", name="load_data")
def load_data():   
    """Load Pycaret Airline dataset."""    
    return get_data("airline")


```

Reply:
I am not familiar with this. Could you take a stab at this in your next PR? I will be happy to learn from that and incorporate it in my future PRs.